docs/1.getting-started/6.server-directory.md

```diff
@@ -0,0 +1,73 @@
+---
+navigation.icon: uil:laptop-connection
+description: Using server directory to create middlewares, plugins and API easily.
```

Comment:
I feel like this should be less about the server _directory_ which is covered in https://nuxt.com/docs/guide/directory-structure/server and more about the _concepts_ of how Nuxt + Nitro inter-operate.

What do you think?